packaging/hudi-metaserver-server-bundle/pom.xml

```diff
@@ -6,20 +6,66 @@
         <artifactId>hudi</artifactId>
         <groupId>org.apache.hudi</groupId>
         <version>0.13.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>hudi-metaserver-server-bundle</artifactId>
 
     <properties>
         <main.basedir>${project.parent.basedir}</main.basedir>
+        <h2.version>1.4.200</h2.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.apache.hudi</groupId>
             <artifactId>hudi-metaserver-server</artifactId>
             <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>${h2.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <!-- Logging -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>${log4j2.version}</version>
```

Comment:
so the logging issue resolved? care to share the findings?

Reply:
Yes. Add these log related dependencies